ament_tools/verbs/build_pkg/cli.py

```diff
@@ -185,6 +185,13 @@ def add_arguments(parser):
         default=False,
         help='Use symlinks instead of copying files wherever possible',
     )
+    parser.add_argument(
+        '--python-interpreter',
+        default=sys.executable,
+        help='Use a different Python interpreter for invoking subprocesses'
+             " (default '%s')" % sys.executable,
```

Comment:
nitpick: using different quote types on strings that are implicitly concatenated looks weird, I'd recommend changing the first lint to use `"` too.